_posts/2019-02-25-the-western-grid.md

```diff
@@ -0,0 +1,136 @@
+
+The Gunter Chain made its way across the Atlantic with the early colonists. It was so popular that the acre, noted in the Articles of Confederation (1781), was declared the standard unit of land measurement for the new nation.
+
+## Standard Units of Measure (the acre)
+{: .text-left}
+
+<div class="caption">
+  <img src="{{ "/images/NorthwestTerritory.png" | prepend: site.baseurl }}" alt="Northwest Territory" />
+  <p class="caption-text">Fig 3: Northwest Territory</p>
+</div>
+
+Okay, let’s jump ahead to 1783 and the Treaty of Paris. At this point in American history, the thirteen original colonies just pulled off the seemingly impossible and defeated the British Empire, ending their colonial rule. This treaty marked the official end of the Revolutionary War and the beginning of independent American governance. In addition to the thirteen colonies, a large swath of land referred to as the Northwestern Territory (Fig 3), was ceded to the United States. This was good news to a new nation carrying a sizable amount of wartime debt. Fortunately, the largely, uninhabited Northwest Territory could be divided and sold off in sections (parcels) to generate revenue for the new nation. Though, first, this land had to be surveyed.
+
+The early American Congress further solidified the acre as the official unit of land measurement with the Land Ordinance of 1785 and the resulting North West Ordinance of 1787. These ordinances provided the framework of the public lands and the procedures for organizing territorial lands west of the Application Mountains.
+
+## Public Land Survey System
+{: .text-left}
+
+I know, I know… that was probably more history than you were looking for, but really, that’s the annotated, crash-course for understanding the basis of the present-day Public Land Survey System (PLSS) (Fig 4). This system was originally proposed by Thomas Jefferson, and was mandated by Congress to oversee the cadastral surveys of the public lands. Essentially, it was set up to facilitate the transfer of Federal lands to private citizens. The Bureau of Land Management (BLM) is the official record keeper of the surveys. Over the past 200 years, almost 1.5 billion acres have been surveyed into townships (and ranges) and sections. These surveys were conducted primarily west of the original thirteen colonies and north of Texas (Texas has Spanish roots and Spanish land grants).  Presently, areas of Alaska are still being surveyed.
+
+<div class="caption">
+  <a href="{{ "/images/principal_meridians.png" | prepend: site.baseurl }}" title="click to see the full sized image">
+    <img src="{{ "/images/principal_meridians_sm.png" | prepend: site.baseurl }}" alt="PLSS Principal Meridians">
+  </a>
+  <p class="caption-text">Fig 4: PLSS Principal Meridians</p>
```

Comment:
Any chance we could make figure 4 on the left and fig 5 on  the right? That would seem to make more sense logically.

Reply:
@steveoh do you know how to do that?  I spent a bunch of time trying to figure that, too and I got nowhere.  I settled on just leaving it.

Reply:
yes I know how to do that.